class.jetpack.php

```diff
@@ -6354,4 +6354,13 @@ function jetpack_user_col_style() {
 			</style>
 		<?php }
 	}
+
+	/**
+	 * Return blog locale as a two-character string
+	 *
+	 * @return string The blog's locale, e.g. 'en' or 'it'
+	 */
+	public static function get_blog_locale() {
```

Comment:
I think this will not work as expected. The wpcom local should be calculated something like the following 
https://github.com/Automattic/jetpack/blob/master/_inc/lib/admin-pages/class.jetpack-react-page.php#L206
```
$localeSlug = explode( '_', get_locale() );
return $localeSlug[0]; 
```

Since you can't assume that the wpcom is only the first 2 characters. Since there are languages such as `Alemannisch` that use more then 2 characters to identify the language. 

But maybe I am completely wrong then you can just ignore me :)